### PR TITLE
Create field for "authors"

### LINKED ISSUE #70: Create field for "authors"
This would help to list a author field per resource (Person, Organisation) or add a link to reference to the owner of the document/OER

### PR DESCRIPTION
**Pull request by saisilinus**
Adds a list of authors and updates resources with the author
Creates a list of authors when a module is dragged
Fixes other small bugs like the positioning of the time breaks and license

Fixes #70

### CODE REVIEW

**Comment by timmwille** [comment]
@saisilinus I saw you created a whole new directory for the authors, interesting approach though my worry is, this adds a lot of overhead for maintaining and adding content so I think we should for now keep it inside the resources files !?

**Comment by saisilinus** [comment]
> @saisilinus I saw you created a whole new directory for the authors, interesting approach though my worry is, this adds a lot of overhead for maintaining and adding content so I think we should for now keep it inside the resources files !?

Ok. I tried keeping the author information inside the resource files initially but there was a lot of repetition especially if someone wanted to add multiple authors. I changed to the current approach to make it easy to update/add a resource i.e. for an author that already exists, all a user has to do is add the author's shortname to the resource's list of authors. If it's a new author, then, sure, you have to add the author to the `_authors` directory first. Do you want me to get rid of the `_authors` folder directory and just add the authors directly to each resource?

**Comment by timmwille** [comment]
lets keep this branch and PR as WIP (work in progress) and try a simpler approach first 



```
resource 
          author: '[Timm Wille](https://openculture.agency/team/)' 
```

- add `author:` (single form usually so we add the comment `#add the author/organisation name in here, use [markdown](URL) formatting to link to website/reference. You can add also multiple authors via '[author 1](link)' 'author 2'].` to .template for resources
- make markdown format possible right now `[author](link)` 
- [ ] is it possible to add multiple authors via `'[author 1](link)' 'author 2'].`? or does it require a different format with comma separation?
- prepare option for later to add `['r0g' 'timm'] multiple author files referenced via ID similar to resources in a module (I like the idea, it is just right now a bit tricky to maintain, but we might go with it at a later stage)

**Comment by saisilinus** [comment]
> lets keep this branch and PR as WIP (work in progress) and try a simpler approach first
> 
> ```
> resource 
>           author: '[Timm Wille](https://openculture.agency/team/)' 
> ```
> 
> * add `author:` (single form usually so we add the comment `#add the author/organisation name in here, use [markdown](URL) formatting to link to website/reference. You can add also multiple authors via '[author 1](link)' 'author 2'].` to .template for resources
> * make markdown format possible right now `[author](link)`
> * [ ]  is it possible to add multiple authors via `'[author 1](link)' 'author 2'].`? or does it require a different format with comma separation?
> * prepare option for later to add `['r0g' 'timm'] multiple author files referenced via ID similar to resources in a module (I like the idea, it is just right now a bit tricky to maintain, but we might go with it at a later stage)

Ok. I will try single author markdown for now and see if it's possible to add multiple authors

**Comment by saisilinus** [comment]
> lets keep this branch and PR as WIP (work in progress) and try a simpler approach first
> 
> ```
> resource 
>           author: '[Timm Wille](https://openculture.agency/team/)' 
> ```
> 
> * add `author:` (single form usually so we add the comment `#add the author/organisation name in here, use [markdown](URL) formatting to link to website/reference. You can add also multiple authors via '[author 1](link)' 'author 2'].` to .template for resources
> * make markdown format possible right now `[author](link)`
> * [ ]  is it possible to add multiple authors via `'[author 1](link)' 'author 2'].`? or does it require a different format with comma separation?
> * prepare option for later to add `['r0g' 'timm'] multiple author files referenced via ID similar to resources in a module (I like the idea, it is just right now a bit tricky to maintain, but we might go with it at a later stage)

How does it look?

**Comment by timmwille** [comment]
@saisilinus I will check asap and send you feedback, might take me up to Monday

**Comment by timmwille** [comment]
Hey @saisilinus can you do me a favour and bring your branch up to date with the other fixes and feature branches you created? (e.g. initiateTimeEdit is still in and some duplicates are challenging me a bit during the review)

**Comment by saisilinus** [comment]
Sure.

**Comment by saisilinus** [comment]
> Hey @saisilinus can you do me a favour and bring your branch up to date with the other fixes and feature branches you created? (e.g. initiateTimeEdit is still in and some duplicates are challenging me a bit during the review)

It should be ok now

**Comment by timmwille** [comment]
@saisilinus I finally found time to test your approach together with @walthierer, quick feedback:

- [ ] 1. your suggestion of "References" won, so I think my ideas to call it "Authors" or "Credits" are down :) if you kindly would change this back to the word "References" 
- [ ] 2. As many Authors could come to play wording for the preview section of each resource should be "Author(s): author1, author2"
- [ ] 3. in the Print view small changes: License in brackets and/or with "License:" in front (also is the dot behind the resource name important? → optional change)

**Review by timmwille — APPROVED** [review]
apart from maybe commenting out the webrick (only needed for local testing, but can also be included on development for now, will set quiet for main branch later) and solving some conflicts looks really good

File: training.html [hunk 1]
```diff
@@ -319,7 +324,7 @@ <h2>Summary</h2>
             
         </div>
         <div class="resource-overview">
-            <h2>Resources</h2>
+            <h3>Resources</h3>
```
[the remark this answers is not in the document]
Comment (timmwille) [reply training.html:371]: just making sure all headings are the same?

[the remark this answers is not in the document]
Comment (saisilinus) [reply training.html:373]: > just making sure all headings are the same?

Yes. I wanted all three to be H3

[the remark this answers is not in the document]
Comment (saisilinus) [reply training.html:373]: > what exactly does this doe? also the code below, is that for the print view?

Which code? I didn't change anything on the Resources section aside from the heading

**Pull request merged**